tsconfig.esm.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.build.json",
+  "compilerOptions": {
+    "module": "es6",
+    "moduleResolution": "node"
```

Comment:
Keeping the `"moduleResolution": "node"` so it can be converted to `"moduleResolution": "node12"` with TSC 4.6